packages/@coorpacademy-components/src/template/my-learning/index.js

```diff
@@ -138,7 +138,11 @@ const MyLearning = (props, context) => {
   const [skillFocusSelectedOnChart, setSkillFocusSelectedOnChart] = useState(undefined);
   const [searchValue, setSearchValue] = useState('');
   const [searchResults, setSearchResults] = useState(
-    skills.sort((a, b) => skillsInformation[b].stats.score - skillsInformation[a].stats.score)
+    skills.sort((a, b) => {
+      const scoreA = skillsInformation[a] ? skillsInformation[a].stats.score : 0;
+      const scoreB = skillsInformation[b] ? skillsInformation[b].stats.score : 0;
+      return scoreB - scoreA;
+    })
```

Comment:
you could use [sortBy](https://lodash.com/docs/4.17.15#sortBy)
```suggestion
    _.sortBy(skillRef => getOr(0, [skillRef, 'stats', 'score'], skillsInformation), skills)
```